Editor/CoreAPI/SettingsKeys.cs

```diff
@@ -16,5 +16,6 @@ public static class SettingsKeys
         public const string DeploymentBuildFilePath = "DeploymentBuildFilePath";
         public const string DeploymentBuildFolderPath = "DeploymentBuildFolderPath";
         public const string WasSettingsWindowShown = "WasSettingsWindowShown";
+        public const string SelectedProfile = "SelectedProfile";
```

Comment:
Is there a reason this is different from `CurrentProfileName`? Is this attempting to do something different?